modules/src/core/ics23_commitment/specs.rs

```diff
@@ -1,6 +1,7 @@
 use crate::prelude::*;
 use ibc_proto::ics23::ProofSpec as ProtoProofSpec;
 use ics23::ProofSpec;
```

Comment:
Just FYI, with regard to #853 - this file (i.e. `modules/src/core/ics23_commitment/specs.rs`) is the only place where we use the `ics23` crate. We only use it to get the definitions for the proof-specs for cosmos and to provide `convert::From` impls for  our domain type (i.e. `ProofSpecs`). For all other cases, we use the `ibc_proto::ics23` defs because we need to derive serde and other traits for these types.